packages/react/src/components/scheduler-month-agenda/scheduler-month-agenda.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Strange name of the component. Why month agenda? It seems customers can use it for different views